tests/unit/test_external_user.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you add also tests when an uploader can share a restricted video with an external user? That is crucial so we can test the workflow needed for the lectures.